src/foundry/client/pixi/placeables/drawing.d.ts

```diff
@@ -1,204 +1,184 @@
 import { ConfiguredDocumentClass, ConfiguredDocumentClassForName } from "../../../../types/helperTypes";
 import { DocumentModificationOptions } from "../../../common/abstract/document.mjs";
 
+export {};
+
 declare global {
   /**
    * The Drawing object is an implementation of the PlaceableObject container.
    * Each Drawing is a placeable object in the DrawingsLayer.
    */
-  class Drawing extends PlaceableObject<InstanceType<ConfiguredDocumentClass<typeof DrawingDocument>>> {
+  class Drawing extends PlaceableObject<any> {
```

Comment:
Why was this changed?

Reply:
This was the instantiation depth bug.

Reply:
Ah, put a comment with the original type and a todo to put it back then, please.